src/www/ui/api/documentation/openapi.yaml

```diff
@@ -3286,6 +3334,15 @@ components:
             2: Advisor
           minimum: 0
           maximum: 2
+    ClearingDecision:
+      description: Clearing decision information
+      type: object
+      properties:
```

Comment:
Please add the missing `globalDecision` property to the model.

Reply:
Updated the changes